File: 2006Scape Server/src/main/java/com/rs2/integrations/discord/commands/Hiscores.java
package com.rs2.integrations.discord.commands;

import com.rs2.Constants;
import com.rs2.integrations.discord.JavaCord;
import org.javacord.api.entity.message.Message;
import org.javacord.api.event.message.MessageCreateEvent;
import org.javacord.api.listener.message.MessageCreateListener;

public class Hiscores implements MessageCreateListener {
    @Override
    public void onMessageCreate(MessageCreateEvent event) {
        Message message = event.getMessage();
        if (message.getContent().equalsIgnoreCase(JavaCord.commandPrefix + " hiscores") || message.getContent().equalsIgnoreCase(JavaCord.commandPrefix + " highscores")) {
            event.getChannel().sendMessage(Constants.WEBSITE_LINK + "/hiscores.html");
        }
    }
}

File: 2006Scape Server/src/main/java/com/rs2/integrations/discord/commands/Hiscores.kt
package com.rs2.integrations.discord.commands

import com.rs2.GameConstants
import com.rs2.integrations.discord.JavaCord
import org.javacord.api.event.message.MessageCreateEvent
import org.javacord.api.listener.message.MessageCreateListener

class Hiscores : MessageCreateListener {
    override fun onMessageCreate(event: MessageCreateEvent) {
        val message = event.message
        if (message.content.equals(JavaCord.commandPrefix + " hiscores", ignoreCase = true) || message.content.equals(
                JavaCord.commandPrefix + " highscores",
                ignoreCase = true
            )
        ) {
            event.channel.sendMessage(GameConstants.WEBSITE_LINK + "/hiscores.html")
        }
    }
}